test/config.test.js

```diff
@@ -756,11 +965,27 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
         });
         assert.equal(SessionRecipe.getInstanceOrThrowError().config.cookieDomain, undefined);
         assert.equal(SessionRecipe.getInstanceOrThrowError().config.cookieSameSite, "lax");
-        assert.equal(SessionRecipe.getInstanceOrThrowError().config.cookieSecure, false);
+        assert.equal(SessionRecipe.getInstanceOrThrowError().config.cookieSecure, true);
         assert.equal(
             SessionRecipe.getInstanceOrThrowError().config.refreshTokenPath.getAsStringDangerous(),
             "/auth/session/refresh"
         );
         assert.equal(SessionRecipe.getInstanceOrThrowError().config.sessionExpiredStatusCode, 401);
     });
+
+    it("testing getTopLevelDomain function", async function () {
```

Comment:
Make sure that you have covered all the domain types based on the normaliseDomainURL tests